ci/nix/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# This image is only for extracting and cleaning up Android SDK
+FROM ubuntu:18.04 AS android_sdk
+
+ARG ANDROID_SDK_VERSION
+ARG SDK_PLATFORM_VERSION
+ARG SDK_BUILD_TOOLS_VERSION
+
+# names of packages to install with sdkmanager
+ENV SDK_PACKAGES emulator patcher;v4 \
+                 platforms;android-26 \ 
+                 platforms;${SDK_PLATFORM_VERSION} \
+                 build-tools;${SDK_BUILD_TOOLS_VERSION} \
+                 extras;android;m2repository extras;google;m2repository
+
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
```

Comment:
The `PATH` here was for the sake of Jenkins, if this is an intermediate image for preparing NDK and SDK this shouldn't be necessary.